release-notes/DxMs/supportassistant_change_log.md

```diff
@@ -4,6 +4,14 @@ uid: supportassistant_change_log
 
 # Support Assistant change log
 
+#### 1 September 2025 - Fix - SupportAsistant 1.7.7 - CDMR reporting & usage offloading issues
+
+An issue, introduced in SupportAsistant 1.7.4 caused the CDMR reporting for proactive support and usage offloading to fail in case the server had no direct internet access instead of routing the request via the CloudGateway. This issue has been resolved and the affected versions have been unlisted.
```

Comment:
```suggestion
An issue introduced in SupportAssistant 1.7.4 caused the CDMR reporting for proactive support and usage offloading to fail in case the server had no direct internet access instead of routing the request via the CloudGateway. This issue has been resolved and the affected versions have been unlisted.
```